web/lang/ru_ru.php

```diff
@@ -74,218 +75,221 @@
     '24BitColour'          => '24 битный цвет',
     '32BitColour'          => '32 битный цвет',          // Added - 2011-06-15
     '8BitGrey'             => '256 оттенков серого',
-    'Action'               => 'Action',
-    'Actual'               => 'Действительный',
-    'AddNewControl'        => 'Add New Control',
+    'Action'               => 'Действие', 
+    'Actual'               => '1х',
```

Comment:
I'm not wild about this line.  The word Actual may be used in places where 1x doesn't make any sense.